Controller/Component/WizardComponent.php

```diff
@@ -157,6 +157,13 @@ class WizardComponent extends Component {
 	public $nestedViews = false;
 
 /**
+ * Holds the root of the session key for data storage.
+ *
+ * @var string
+ */
+	public $sessionRootKey = 'Wizard';
```

Comment:
Yeah, that  works with both the component and the helper. Nothing extra should be done in order to pass this setting to the helper.